security-privacy-questionnaire.md

```diff
@@ -0,0 +1,82 @@
+# Security and Privacy Questionnaire
+
+This document answers the [W3C Security and Privacy
+Questionnaire](https://www.w3.org/TR/security-privacy-questionnaire/) for the
+Picture-in-Picture specification.
+
+**Does this specification deal with personally-identifiable information?**
+
+No.
+
+**Does this specification deal with high-value data?**
+
+No.
+
+**Does this specification introduce new state for an origin that persists across
+browsing sessions?**
+
+No.
+
+**Does this specification expose persistent, cross-origin state to the web?**
+
+No.
+
+**Does this specification expose any other data to an origin that it doesn’t
+currently have access to?**
+
+No.
+
+**Does this specification enable new script execution/loading mechanisms?**
+
+No.
+
+**Does this specification allow an origin access to a user’s location?**
+
+No.
+
+**Does this specification allow an origin access to sensors on a user’s
+device?**
+
+No.
+
+**Does this specification allow an origin access to aspects of a user’s local
+computing environment?**
+
+On Android, it allows an origin to know if user has toggled 
+off "Allow picture-in-pucture" system setting.
```

Comment:
Can we do something more generic such as `pictureInPictureEnabled` will reflect the state of the Picture-in-Picture" setting on the system or user agent.

Reply:
Done.